### PR TITLE
LTP/install: Fix URL on requested addon (module) for SLE

### PR DESCRIPTION
**Pull request by pevik**
Desktop-Productivity module does not exist any more (bsc#1078799)

- Verification run: http://quasar.suse.cz/tests/471
- Fixing failure: https://openqa.suse.de/tests/1441996#step/install_ltp/12

### CODE REVIEW

File: tests/kernel/install_ltp.pm [hunk 1]
```diff
@@ -30,22 +30,23 @@ sub scc_we_enabled {
...
-}
-
 sub add_we_repo_if_available {
     # opensuse (doesn't have extensions) or SLE as registered product
     if (check_var('DISTRI', 'opensuse') || scc_we_enabled) {
         return;
     }
+    my $ar_url;
+    my $we_repo = get_var('REPO_SLE_WE15_POOL');
+    if ($we_repo and check_var('DISTRI', 'sle') and sle_version_at_least('15')) {
```
Comment (richiejp) [inline tests/kernel/install_ltp.pm]: `&&` not `and`

Comment (SergioAtSUSE) [reply tests/kernel/install_ltp.pm]: Why? if && is wanted, why is it not added to the code verification on travis?

Comment (richiejp) [reply tests/kernel/install_ltp.pm]: It has weaker precedence which is different to the `and` operator in most programming languages. It should be disallowed for people's sanity, but someone has to do a code transformation to convert all the `and`s to `&&` first taking the difference in precedence into account.

Comment (pevik) [reply tests/kernel/install_ltp.pm]: I'm sorry for keeping forgetting about it. I learned using `and` many years ago. I'll be better next time.
@SergioAtSUSE If you create perlformat rule I volunteer to fix the occurrences.

**Pull request merged**